### PR TITLE
Installing theme visual indication.

### PR DESCRIPTION
**Pull request by budzanowski**
### Info
This is for WP.com theme installation on Jetpack site. Because it may take significant amount of time(for example flaky connection) for theme to install we need to give user visual clue that the process is ongoing.

### How it looks
Theme gets grayed out and pulsating dot appears.
installing

### Testing open Jetpack site and click `Try&Customize` from the ellipsis menu.

### CODE REVIEW

File: client/components/theme/index.jsx [hunk 1]
```diff
@@ -137,6 +149,7 @@ const Theme = React.createClass( {
 				<div className="theme__content">
 					{ this.renderHover() }
 					<a href={ this.props.screenshotClickUrl }>
+						{ this.renderInstalling() }
```
Comment (seear) [inline client/components/theme/index.jsx]: I'd be tempted to go with `{ installing && this.renderInstalling() }` here, meaning we could lose an indent level inside `renderInstalling()`, but there is nothing wrong with it as is.

File: client/components/theme/style.scss [hunk 2]
```diff
@@ -158,6 +160,16 @@ $theme-info-height: 54px;
 	height: $theme-info-height;
 }
 
+.theme__installing {
+	background: transparentize($gray-dark, 0.5);
+	width: 100%;
+	left: 0;
+	top: 0;
+	bottom: 54px;
```
Comment (budzanowski) [inline client/components/theme/style.scss]: 54px comes from Theme component CSS. It is just how much space the lower part takes (the one with ellipsis menu) I am changing it to already used `$theme-info-height` variable that holds that value for SSOT. And the value itself is not scaled. Name + ellipsis have always the same height.

**Comment by folletto** [comment]
Excellent work. 👍

**Comment by lancewillett** [comment]
Ready to merge?

**Pull request merged**